app/src/main/java/org/mozilla/focus/ext/Context.kt

```diff
@@ -51,7 +51,7 @@ val Context.installedDate: String
 /**
  * Checks if the current device is a tablet.
  */
-fun Context.isTablet(): Boolean = resources.getBoolean(R.bool.is_tablet)
+fun Context.isTablet(): Boolean = HardwareUtils.isTablet(this)
```

Comment:
I don't see this being used in Fenix (although it has the old method with values in configuration.xml) but I see no reason this could not live in AC. 

We could wait for the project migration to decide what will we do with all the extensions we already have in sour eparate repos (maybe they should live in a single module in AC? 🤔 )